exercises/concept/lasagna/lasagna_test.cpp

```diff
@@ -0,0 +1,74 @@
+#include "lasagna.cpp"
+#ifdef EXERCISM_TEST_SUITE
+#include <catch2/catch.hpp>
+#else
+#include "test/catch.hpp"
+#endif
+
+using namespace std;
+
+TEST_CASE("Preparation time correct") {
+    const int actual = 40;
```

Comment:
This is not strictly needed, but seeing that the student does not yet know about `const`, maybe just do:

```suggestion
    int actual = 40;
```

That way it is more likely that the student understands the test suite.